ci/build_container/build_container_ubuntu.sh

```diff
@@ -31,7 +24,7 @@ update-alternatives --config gcc
 update-alternatives --config g++
 update-alternatives --config gcov
 # Bazel and related dependencies.
-apt-get install -y openjdk-8-jdk
+apt-get install -y openjdk-8-jdk curl
```

Comment:
Why are these not included in the first list above?  I noticed that you shifted to wget for the llvm key because curl is no longer available here.

Reply:
This is part of "revert local changes to reduce diff with upstream".  I originally changed it when adding CMake in 62d70b228c4f4969bd868ced1b34b9d48873d82d, but it was unnecessary "cleanup" that I never upstreamed.

The only change in this file (compared to upstream) is addition of `libglib2.0-dev`.

Sorry for making diffs hard...